### PR TITLE
Added Support, Info, and 404 pages, plus Alphabetical Sorting

### PR DESCRIPTION
**Pull request by YewoMhango**
# Changes
1. For issue #6, the following pages were added:
     - Support
     - Info
     - 404
2. For issue #5, it's now possible to sort hymns in the list alphabetically (using their title) or using the hymn number

### CODE REVIEW

**Comment by OlivierJM** [comment]
Great stuff @YewoMhango I was also thinking that maybe instead of the home page, we default to English hymn and just list them like this https://cis-web-app-pr-58.onrender.com/songs/english/1 
Maybe this can be handled in  a separate issue

**Comment by YewoMhango** [comment]
Oh, okay. Is that really necessary, though?

**Comment by OlivierJM** [comment]
> Oh, okay. Is that really necessary, though?

No it's really not.

File: src/Pages/Support/Support.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                        target="_blank"
+                    >
+                        <img src="/images/PayPal.svg" alt="" />
+                    </a>
+                </p>
+                <p className={styles.disclaimer}>
+                    A One-time Donation or a Monthly Donation is not required to
+                    access any feature of this web app, All features are
+                    available for free and will always be.
+                    {/* The subscription renews automatically each month. You can
```
Comment (OlivierJM) [inline src/Pages/Support/Support.tsx:33]: We can get rid of this.

**Review by OlivierJM — APPROVED** [review]
This is good.

**Pull request merged**